package.json

```diff
@@ -20,15 +20,21 @@
     "url": "git://github.com/edwellbrook/node-tvdb.git"
   },
   "dependencies": {
-    "jszip": "3.1.2",
-    "request": "2.74.0",
-    "xml2js": "0.4.17"
+    "debug": "^2.2.0",
```

Comment:
Not convinced the information being debugged is useful enough to warrant outputting, and an additional dependency. Can we remove this please.